### DISCUSSION TITLE
Is it possible to set block/allow lists?

### DISCUSSION THREAD

**Discussion by liorix1** [kkebo/DNSecure#79] [Q&A]
In NextDNS I can put my own domains in the block and allow lists, is it possible on DNSecure too?
DNSCloak was also able to do it, is this app an alternative to DNSCloak in some way?
Thanks!

**Comment by kkebo** [comment]
No, it's not possible until #47 is fixed.

**Reply by liorix1** [reply]
Are there any news about that?
Not supporting block and allow lists is the only things prevents me from moving to DNSecure!

**Reply by kkebo** [reply]
#47 has been fixed in 1.6.0.
